app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -624,13 +625,16 @@ open class DefaultToolbarMenu(
         }
     }
     private fun getSetDefaultBrowserItem(): BrowserMenuImageText? {
-        val experiments = context.components.analytics.experiments
+        val analytics = context.components.analytics
+        val experiments = analytics.experiments
         val browsers = BrowsersCache.all(context)
 
         return experiments.withExperiment(Experiments.DEFAULT_BROWSER) { experimentBranch ->
             if (experimentBranch == ExperimentBranch.DEFAULT_BROWSER_TOOLBAR_MENU &&
                 !browsers.isFirefoxDefaultBrowser
             ) {
+                analytics.metrics.track(Event.SetDefaultBrowserToolbarMenuShown)
```

Comment:
Just noticed this, sorry -- it would be great if this fired from every branch, not just the toolbar-menu branch; we need to observe this in the control branch as well.

Reply:
Or, rather, that would cover https://github.com/mozilla-mobile/fenix/issues/18855

Reply:
👍🏽